src/Executable/Executable.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Castor\Executable;
+
+use Castor\Container;
+use Castor\Context;
+use Symfony\Component\Process\Process;
+
+use function Castor\context;
```

Comment:
In `src` path, we should never use public function.
It's better to use proper DI.

Could you inject the ContextRegistry in the constructor instead?

Reply:
It's complicated to use the DI here since this class intends to be used by client or plugins, AFAIK we don't support injecting those classes into DI

Reply:
userland code could use function, this is fine. 
For example, in the functions.php we do that:
https://github.com/jolicode/castor/blob/f449806464b4d1a0660e7a9434cbc4ad7703b44f/src/functions.php#L585-L586

People could do that

```php
#[AsTask(name: 'ls', description: 'Run a sub-process and display information about it')]
function ls_task(): void
{
    ls(context(), __DIR__ . '/executable')->run();
```